org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -435,8 +435,9 @@ SsoDeveloper:
       - 'arn:aws:iam::aws:policy/AWSBillingReadOnlyAccess'
       - 'arn:aws:iam::aws:policy/AmazonBedrockFullAccess'
     sessionDuration: 'PT12H'
-    inlinePolicy: >-
-      {
+    inlinePolicy:
```

Comment:
This is a parameter to a template, can we actually use a function here? From the previous failure it looks like it was passed straight through.

Reply:
Let's try...